CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.3.5-dev6
```

Comment:
With local inference in there now, we can probably release a `0.3.6`

Reply:
Changed version to release version in [4e65e6e](https://github.com/Unstructured-IO/unstructured/pull/125/commits/4e65e6e326e871b2e626af12075aacad8e684abe).